srcpkgs/howl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'howl'
+pkgname=howl
+version=0.6
+revision=1
+archs="i686 x86_64 x86_64-musl"
+build_wrksrc=src
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="wget gcc pkg-config gtk+3"
```

Comment:
Remove everything except `pkg-config` as the distribution files are already fetched when building the package. The compiler should not be explicitly be specified in the `hostmakedepends` variable.

Reply:
Right, this should just be `pkg-config`

Reply:
Done.